virtwho.py

```diff
@@ -175,50 +240,67 @@ def run(self):
             return
 
         result = {}
+        report = None
         while not self.terminate_event.is_set():
             # Wait for incoming report from virt backend
             try:
-                report = self.queue.get(block=True, timeout=None)
+                report = self.queue.get_nowait()
```

Comment:
This looks like busy waiting if I'm not mistaken. We'll just check for Jobs in the self.jobs in loop without any waiting. I think we should wait for new event in self.queue for some time and then check all the Jobs in the self.jobs.